### PR TITLE
Update operator image version and channel

### PR DESCRIPTION
**Pull request by swati-nair**
1. Updated defaultChannel to stable-v1
2. Updated version to 3.5.0
3. Also, updated the push_csv.sh script to include the deleting csv. (Consistent with ODLM repo's push_csv.sh)

### CODE REVIEW

File: common/scripts/push_csv.sh [hunk 1]
```diff
@@ -47,12 +47,26 @@ else
...
+function delete_csv() {
+  echo "Delete release ${RELEASE} of package ${QUAY_REPOSITORY} from namespace ${QUAY_NAMESPACE}"
+  curl -H "Content-Type: application/json" \
+      -H "Authorization: ${AUTH_TOKEN}" \
+      -XDELETE https://quay.io/cnr/api/v1/packages/"${QUAY_NAMESPACE}"/"${QUAY_REPOSITORY}"/"${RELEASE}"/helm
+}
+
+#-------------------------------------- Main --------------------------------------#
+delete_csv
+push_csv
```
Comment (Crystal-Chun) [inline common/scripts/push_csv.sh]: Make sure you add an extra line at the end of files.

Comment (swati-nair) [reply common/scripts/push_csv.sh]: done

File: deploy/olm-catalog/ibm-cert-manager-operator/3.5.0/ibm-cert-manager-operator.v3.5.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -441,7 +441,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: ibm-cert-manager-operator
-                image: quay.io/opencloudio/ibm-cert-manager-operator:latest
+                image: quay.io/opencloudio/ibm-cert-manager-operator:3.5.0
                 imagePullPolicy: Always
                 name: ibm-cert-manager-operator
                 resources: {}
```
Comment (Crystal-Chun) [inline deploy/olm-catalog/ibm-cert-manager-operator/3.5.0/ibm-cert-manager-operator.v3.5.0.clusterserviceversion.yaml]: This section represents the `operator.yaml` file in the deploy folder. Make sure it reflects the same changes/information in here.

Comment (swati-nair) [reply deploy/olm-catalog/ibm-cert-manager-operator/3.5.0/ibm-cert-manager-operator.v3.5.0.clusterserviceversion.yaml]: done

**Pull request merged**